ansible/roles/slurm_node/templates/01_slurm.conf.j2

```diff
@@ -0,0 +1,2 @@
+[domain/ikim.uk-essen.de]
```

Comment:
Just curious: Do we not have a variable containing our domain name?
Having this hard-coded in the role doesn't seem good.